documentation/rdme.md

```diff
@@ -8,18 +8,28 @@ category: 5f7ce9e3a5504d0414d024c2
 
 Hello curious raw Markdown reader! 👋
```

Comment:
Should this file be a symlink to the root readme?

Reply:
The top section has some overlap but I think of the root README as being a command reference, whereas this is more of an in-depth "getting started" guide.